### DISCUSSION TITLE
Mercado de T.I em Geral e em Fortaleza CE

### DISCUSSION THREAD

**Discussion by lzhudson** [frontendbr/forum#1133] [Discussão]
Gostaria de levantar uma discussão sobre o mercado de t.i em geral porém um pouco voltado a cidade de Fortaleza no Ceará, pois estou querendo investir tempo e dedicação nesta areá principalmente em uma faculdade. Já sou formado em tecnico em T.I voltado a areá da progamação terminei recentemente o curso adjunto ao ensino médio. Então gostaria que falassem um pouco sobre o mercado no geral e um pouco sobre Fortaleza.
Desde já agradeço e boa noite.

**Comment by cjpatoilo** [comment]
Olá @lzhudson. Atualmente moro em Fortaleza e trabalhei meus 2 primeiros anos como FrontEnd nesta cidade. Então vou tentar resumir minha visão sobre o cenário de Fortaleza e se algo não ficou claro ou se você tiver alguma dúvida a mais, sinta-se a vontade para me adicionar em alguma rede social p/ trocarmos algumas ideias. Meu ID em todas redes sociais: @cjpatoilo. 

Fortaleza é uma cidade incrível. Há boas empresas com produtos desafiadores, ótimos profissionais morando na cidade e uma comunidade que sempre tenta fazer o melhor. Sempre há bons meetups e ótimos eventos ([FrontInFortaleza](http://frontinfortaleza.com.br/), [CEJS](http://www.cejs.com.br/), [DEV IO](https://www.facebook.com/groups/deviofoundation)), além dos grandes eventos que já foram organizados em nossa cidade ([BrazilJS](http://fellipe.com/blog/brazil-js-2011/), [JSConfBrazil](http://2014.jsconfbr.org/)).

Sobre o mercado, nosso cenário não é tão aquecido como de outras cidades do sul e sudeste. Porém isso pode ser um ponto muito positivo por não existir uma grande concorrência para conseguir a primeira oportunidade na área.

ps: Atualmente trabalho para uma empresa de outra cidade de forma remota e assíncrona. Assim como meu exemplo há muitos outros profissionais que trabalham daqui assim. ;)

**Comment by FilhoSergio** [comment]
Boa issue , sou de Fortaleza  e achei bacana conhecer esses meetups , confesso que nao conhecia .

**Comment by cjpatoilo** [comment]
@lzhudson @FilhoSergio o [DEV IO](https://deviofoudantion.org) é um grupo de Fortaleza muito bacana. Falamos praticamente sobre tudo lá, principalmente sobre o mundo Front-End (HTML, CSS & JS). É um espaço para tirar dúvidas, fazer muito networking, compartilhar conhecimento, experiências e informações sobre os eventos e meetups da região. 

Além do [grupo no Facebook](https://www.facebook.com/groups/deviofoundation), temos o [grupo no Slack](https://slack.deviofoundation.org) onde organizamos por channel os assuntos abordados.

Sintam-se a vontade para entrar e convidar quem vocês quiserem. ;)

**Comment by FilhoSergio** [comment]
@cjpatoilo  preciso de convite para entrar no grupo do Slack ? aqui n consegui .

**Comment by cjpatoilo** [comment]
@FilhoSergio https://deviofoundation.herokuapp.com/

**Comment by lipehfama** [comment]
Que felicidade,  saber dessa comunidade em Fortaleza incrível, bem estou em fase de aprendizado, e gostaria de participar desses grupos e ótimo poder conhecer pessoas da sua cidade para trocar conhecimento.
